README.md

```diff
@@ -7,13 +7,18 @@ system.
 
 To run the API locally, run `make dev`. This assumes you have `Makefile`, `Docker` and `docker-compose` installed on your machine.
 
+<br>
+
+![Github Repo Size](https://img.shields.io/github/repo-size/openfoodfacts/openfoodfacts-events?style=for-the-badge&color=aqua)
+
+
```

Comment:
Is this really something we need? The repo is not that big

Reply:
Yes I think it will be better for the contributors to know this fact.